interface/logKappa.h

```diff
@@ -0,0 +1,23 @@
+#ifndef logKappa_h
+#define logKappa_h
+
+template<typename T>
+T logKappa(const T x, const double &kappaHigh, const double &kappaLow) {
```

Comment:
```suggestion
inline double logKappa(double  x, double kappaHigh, double kappaLow) {
```
Why the template? RooFit always deals with `double` anyway. And why the references? Doubles are usually passed by value.